graph/min_cost_b_flow/checker.cpp

```diff
@@ -0,0 +1,207 @@
+#include <vector>
+#include <utility>
+#include <exception>
+
+#include "testlib.h"
+#include "params.h"
```

Comment:
In now, checker.cpp cannot include "params.h" though it works on local and CI. It is clearly the bad point of this judge and we should fix... but anyway, please don't include it, just copy-paste the contents of params.h is okay (I'll fix in later...).